shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -416,6 +423,9 @@ void EmbedderExternalViewEmbedder::SubmitFlutterView(
     GrDirectContext* context,
     const std::shared_ptr<impeller::AiksContext>& aiks_context,
     std::unique_ptr<SurfaceFrame> frame) {
+  // Get the view's cache, or create one if the view doesn't have a cache yet.
+  EmbedderRenderTargetCache& render_target_cache =
```

Comment:
Could add a comment on `EmbedderRenderTargetCache`'s declaration that it is view-specific?